base/uk.ac.stfc.isis.ibex.scriptgenerator/src/uk/ac/stfc/isis/ibex/scriptgenerator/ToyModel.java

```diff
@@ -0,0 +1,23 @@
+package uk.ac.stfc.isis.ibex.scriptgenerator;
+
+import uk.ac.stfc.isis.ibex.model.ModelObject;
+
+public class ToyModel extends ModelObject {
```

Comment:
Javadocs needed otherwise checkstyle will go up (even if this is just a "toy" model)